opentreemap/exporter/tasks.py

```diff
@@ -31,14 +33,18 @@ def extra_select_and_values_for_model(
 
     extra_select = {}
     prefixed_names = []
+    dummy_instance = safe_get_model_class(model)()
 
-    for perm in perms:
+    for perm in (perm for perm in perms if perm.permission_level > 0):
```

Comment:
Consider changing `if perm.permission_level > 0` to `if perm.permission_level >= FieldPermission.READ_ONLY`